app/controllers/administrative-units/administrative-unit/index.js

```diff
@@ -55,12 +79,27 @@ export default class AdministrativeUnitsAdministrativeUnitIndexController extend
   async editContactInfo(event) {
     event.preventDefault();
 
-    if (!this.isSaving) {
-      this.isSaving = true;
+    if (!this.isSavingContact) {
+      this.isSavingContact = true;
 
-      await this.model.psave();
+      let address = await this.model.administrativeUnit.primarySite.get(
+        'address'
+      );
 
-      this.isSaving = false;
+      address.fullAddress =
```

Comment:
I wonder why this is a field in the database when it can just be computed based on the other values. Is that something we can change in the future? (We only set the parts and create a getter / helper to combine them when needed?)

Reply:
In some cases, there's only the full address available.

Reply:
hmmm, is that one of those things we can have someone manually correct?

Reply:
Yes, but not guaranteed yet.